site/_includes/head.html

```diff
@@ -7,6 +7,28 @@
   <meta name="description" content="{{ page.excerpt | default: site.description | strip_html | normalize_whitespace | truncate: 160 | escape }}">
   <meta name="theme-color" content="#F4751E">
 
+  <!-- Schema.org markup for Google+ -->
+  <meta itemprop="name" content="{{ site.title }}">
+  <meta itemprop="description" content="{{ site.description }}">
+  <meta itemprop="image" content="{{ "/assets/images/hnpwa-showcase.png" | relative_url }}">
+
+  <!-- Twitter Card data -->
+  <meta name="twitter:card" content="summary_large_image">
+  <meta name="twitter:site" content="{{ site.twitter_username }}">
+  <meta name="twitter:title" content="{{ site.title }}">
+  <meta name="twitter:description" content="{{ site.description }}">
+  <meta name="twitter:creator" content="{{ site.twitter_username }}">
+  <!-- Twitter summary card with large image must be at least 280x150px -->
+  <meta name="twitter:image" content="{{ "/assets/images/hnpwa-showcase.png" | relative_url }}">
```

Comment:
It looks like the Twitter card also supports `twitter:image:alt` for accessibility. Could we add that in?